### PR TITLE
RNN model correction

### PR DESCRIPTION
**Pull request by ronitd**
RNN model's default parameters contain stride. There are 2 convolutional layers each with a stride of 2. So for num_feature = 80, get reduced to 80 ->40 ->20. Updated the code to integrate it.

### CODE REVIEW

File: models.py [hunk 1]
```diff
@@ -280,9 +281,10 @@ def __init__(
             if dropout > 0:
                 convs.append(torch.nn.Dropout(dropout))
             in_channels = out_channels
-
+            dilation = dilation if dilation else 1
+            h_out = torch.floor((h_out + 2*padding - dilation*(kernel - 1) - 1)/stride + 1)
```
Comment (awni) [inline models.py:285]: There is no option for `dilation` and we always pad by half the kernel size on each side (L270) so this can be probably be simplified to:

`h_out //= stride`

**Review by awni — CHANGES_REQUESTED** [review]
Thanks for fixing this, let's simplify as per comment below (I don't think we have plans to add dilation and user defined padding in the near future..)

**Comment by ronitd** [comment]
Made the changes.

**Comment by awni** [comment]
Thank you!
